components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -1947,7 +2020,7 @@ else if (unit.equals("m")) {
             if (ms.sizeZ == 1) {
               ms.sizeZ = len;
               bytesPerAxis.put(nBytes, "Z");
-              physicalSizeZ = (physicalLen * len) / (len - 1);
+              physicalSizeZ = physicalLen;
```

Comment:
Apologies for being late to the discussion here.  The original rationale for dividing ```physicalSizeZ``` by ```len - 1``` but dividing ```physicalSize{X,Y}``` by ```len``` is that we typically model the physical Z size as the physical distance between two Z sections, or the distance the stage has to move (i.e. the plane itself has 0 thickness).  Physical X and Y sizes are typically modeled as the width and height of each pixel, not the space between two pixels.  As an example, a 1x1 image with 1 Z plane would typically have non-0 physical width and height, but physical Z size of 0.  With the current logic, the same 1x1x1 image would have all 0 physical sizes.